docs/using.rst

```diff
@@ -35,16 +35,20 @@ To install and run the client you just need to type:
 ``letsencrypt``.  ``letsencrypt-auto`` is a wrapper which installs virtualized
 dependencies and provides automated updates during the beta program)
 
-.. warning:: Please do **not** use ``python setup.py install`` or ``sudo pip install`.
-             Those mode of operation might corrupt your operating system and is
-             **not supported** by the Let's Encrypt team!
-
 The ``letsencrypt`` commandline tool has a builtin help:
 
 .. code-block:: shell
 
    ./letsencrypt-auto --help
 
+.. warning:: Please do **not** use ``python setup.py install`` or
+   ``python pip install .``. Please do **not** attempt the
+   installation commands as superuser/root and/or without virtual
+   environment, e.g. ``sudo python setup.py install``, ``sudo pip
+   install``, ``sudo ./venv/bin/...``. These modes of operation might
```

Comment:
Although that's slightly confusing since `letsencrypt-auto` uses `sudo` internally and will prompt you for your `sudo` password. Thoughts @kuba, @pde?